src/Javis.jl

```diff
@@ -253,6 +259,8 @@ function render(
     postprocess_frames_flow = identity,
     postprocess_frame = default_postprocess,
 )
+
+    STARTED_RENDERING[1] = true
```

Comment:
- changed STARTED_RENDERING to CURRENTLY_RENDERING

- didn't move `CURRENTLY_RENDERING[1] = false` inside `empty_CURRENT_variables` because the latter is not called if `pathname` is empty, I don't know if there is a reason for that. if not I can move it inside it and make it a bit cleaner.

Reply:
I think we should call it in all cases.